server/package.json

```diff
@@ -34,6 +34,7 @@
     "sequelize-cli": "^6.2.0",
     "socket.io": "^4.2.0",
     "tmp": "^0.1.0",
-    "uuid": "^3.4.0"
+    "uuid": "^3.4.0",
+    "xmldom": "^0.6.0"
```

Comment:
is this required?

Reply:
Not sure how it got there, it is removed

Reply:
When removed it is breaking added back